test/parallel/test-util-inspect.js

```diff
@@ -1812,3 +1812,11 @@ assert.strictEqual(
     '{ foo: [Getter/Setter] Set { [ [Object], 2, {} ], ' +
       "'foobar', { x: 1 } },\n  inc: [Getter: NaN] }");
 }
+
+{
+  const date = new Date('Sun, 14 Feb 2010 11:48:40 GMT');
+  const isoString = date.toISOString();
+  Object.setPrototypeOf(date, null);
+
+  assert.strictEqual(util.inspect(date), '[Date: null prototype] ' + isoString);
+}
```

Comment:
I suggest to move this up into that test part https://github.com/nodejs/node/pull/25144/files#diff-c1778ece12049a0e08c6f646c02c10d7R1656. There are a couple extra checks and only a single line of code has to be added to test this properly in that case :)